tools/pkg/platforms/centos7/files/build/mongooseim.spec

```diff
@@ -1,31 +1,45 @@
-%undefine _missing_build_ids_terminate_build
-%define _libdir /usr/lib/mongooseim
-%define _etcdir /etc
-%define _bindir /usr/bin
-%define _vardir /var
-
-Summary: %{_summary}
-Name: %{_name}
-Version: %{_vers}
-Release: %{_release}
-Source: %{name}_%{version}.tar.gz
-License: %{_license}
-Group: "Development/Languages"
-BuildArch: @ARCH@
-BuildRoot: %{_topdir}/BUILDROOT/%{name}-%{version}.@ARCH@
-BuildRequires:  esl-erlang >= 21.3, esl-erlang <= 22.3
-BuildRequires:  expat-devel
-BuildRequires:	chrpath
-Requires(pre): shadow-utils
-
-Vendor: %{_vendor}
+Name:              mongooseim
+Version:           %{version}
+Release:           %{release}
+Summary:           A mobile messaging platform with focus on performance and scalability
+
+Group:             Development/Languages
+License:           GPLv2
+URL:               http://packages.erlang-solutions.com
+Packager:          Erlang Solutions Ltd.
+Source0:           mongooseim.service
+
+
+BuildArch:         %{architecture}
+
+BuildRequires:     esl-erlang >= %{esl_erlang_ver_down}, esl-erlang <= %{esl_erlang_ver_up}
```

Comment:
Is it required to specify minimum and maximum Erlang version? Can we specify only the min version?